blueprints/ember-frost-button/index.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  description: '',
+  normalizeEntityName: function () {},
+
+  /**
+    Installs specified packages at the root level of the application.
+    Triggered by 'ember install <addon name>'.
+
+    @returns {Promise} package names and versions
+  */
+  afterInstall: function () {
+    return this.addAddonsToProject({
+      packages: [
+        {name: 'ember-frost-theme', target: '^1.0.3'},
+        {name: 'ember-frost-icons', target: '^1.4.0'}
```

Comment:
I see that `css-core` was in `dependencies` and now is in `devDependencies` but is not here. Was that intentional?